tiflash-610-upgrade.md

```diff
@@ -0,0 +1,83 @@
+---
+title: TiFlash v6.1.0 升级帮助
+summary: 了解升级 TiFlash 至 v6.1.0 时的注意事项。
+---
+
+# TiFlash v6.1.0 升级帮助
+
+本文主要介绍 TiFlash 升级前后具体功能模块的变化，以及推荐的应对方法。
+
+如需了解标准升级流程，请参考如下文档：
+
+- [使用 TiUP 升级 TiDB](/upgrade-tidb-using-tiup.md)
+- [使用 TiDB Operator 升级 TiDB](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/upgrade-a-tidb-cluster)
+
+## 常见升级策略
+
+不推荐跨大版本升级。请先升级至 v5.4.x 或 v6.0.0，然后再升级至 v6.1.0。
+
+### 升级 v4.x.x 至 v5.x.x
+
+v4.x.x 已近品周期尾声，请尽早升级到主流版本。商业客户请联系售后和客服。
+
+### 升级 v5.x.x 至 v6.0.0
+
+v6.0.0 作为非 LTS 版本，不会推出后续的 bug 修复版，建议商业客户慎用。尽可能使用 v6.1.0 及之后的 LTS 版本。
+
+### 升级 v5.x.x 至 v6.1.0
+
+#### <a name="proxy"></a>TiFlash Proxy
+
+TiFlash 在 v6.1.0 对 Proxy 做了升级（与 TiKV v6.0.0 对齐）。该版本升级了 RocksDB 版本，在升级过程中会自动将数据格式转换为新版本。
+
+正常升级时，不会有明显风险，但是，如果你有特殊需求，请注意：v6.1.0 降级到之前的任意低版本时，会无法解析新版 RocksDB 配置，从而导致 TiFlash 重启失败。请做好升级验证工作，并尽可能准备应急方案（确保 TiKV 数据可用，并预估重新同步数据可能造成的影响）。
+
+##### 测试环境及特殊回退需求下的对策
+
+确保相应表中 TiKV 副本的数据可用，强制缩容 TiFlash 节点，并重新同步数据。操作步骤详见[用户手册](/scale-tidb-using-tiup.md#缩容-tiflash-节点)。
```

Comment:
The TiKV replica should always be available. We don't need to mention it here, otherwise, the users will confuse.
```suggestion
强制缩容 TiFlash 节点，并重新同步数据。操作步骤详见[用户手册](/scale-tidb-using-tiup.md#缩容-tiflash-节点)。
```

Reply:
LGTM, please merge